src/Callback.php

```diff
@@ -88,11 +74,18 @@ public function init(): void
             $this->urlTrigger = $this->name;
         }
 
-        if ($this->postTrigger === true) {
-            $this->postTrigger = $this->name;
-        }
+        $this->app->stickyGet($this->urlTrigger);
+    }
 
-        $this->appSticky ? $this->app->stickyGet($this->urlTrigger) : $this->owner->stickyGet($this->urlTrigger);
+    public function setUrlTrigger(string $trigger)
+    {
+        $this->urlTrigger = $trigger;
+        $this->app->stickyGet($this->urlTrigger);
```

Comment:
I belive this concept is wrong, here is why, imagine the following structure:
elem
-> eleme (this set stickyGet)
elem

if the last element does something with the URL, it will still add the sticky parameter

I think we should instead introduce sticky parameters per View element and then when we build URL for View, collect sticky parameters from the parents/owners.

Reply:
Not sure I follow what you mean. Settings stickyGet to App vs View will only ensure that the get argument will be set in URL, no matter which View required it. Thus ensuring the callback will run.

Reply:
I named elements in my example.

Now consider this:
- Form A callback is triggered
- and wants to return rendered Form B

now form B will have sticky GET from Form A

That is definitelly wrong - sticky GET should not be global, but per tree item and only self and parent items should contribute to the sticky GET params used for a new URL